021-IdentityForApps/README.md

```diff
@@ -1,8 +1,112 @@
+
+- Enrich the claims that you collect about a user during the sign up process by calling out to a custom REST API
+
+Challenge 6: **[Add Conditional Access To Your Tenant](Student/06-conditional-access.md)**
+
+- Create and enforce Conditional Access policies in your tenant such as enforcing MFA (including Microsoft Authenticator) and detecting Risky Login Behavior
+
+Challenge 7: **[Admin the B2C Tenant with MS Graph](Student/07-admin-graph.md)**
+
+- Use the MS Graph API to query your B2C tenant. Also use the Graph API to update various objects in your B2C tenant, such as policies, keys, and identity providers.
+
+Challenge 8: **[Prepare Environment for Custom Policies](Student/08-prepare-ief.md)**
+
+- We'll need custom policies, so let's get things ready. Apply the Trust Framework and also create an OIDC IdP for your external IdP
+
+Challenge 9: **[Creating Custom Policies](Student/09-custom-policy.md)**
+
+- Implement a custom policy for Sign In that will call to your custom REST API to perform claims enrichment for users that signed up prior to Challenge 5. Also, we'll break the Sign Up and Sign In policy to be just a Sign Up policy.
+
+Challenge 10: **[Tracking a User's Journey in a Policy](Student/10-appinsights.md)**
+
+- Enable App Insights in your custom policy so you can track a user through the various steps in the Orchestration. Add custom events to your Orchestration and track them in App Insights.
+
+Challenge 11: **[Parameterize Your Custom Policies](Student/11-parameterize.md)**
+
+- Take your custom policies and parameterize the values that could change from environment to environment, and use the B2C extension to VS Code to generate environment-specific policy files.
+
+Challenge 12: **[Monitoring Your Tenant](Student/12-monitor.md)**
+
+- Monitor your B2C tenant by combining logs and app insights logs
+
+## Challenge Flow
+
+This Identity Hack can be broken into two smaller hacks: one focusing just on User Flows and the other focusing on Custom Policies. If you want a hack focused on one of these tracts, this diagram illustrates the common challenges and the tract-specific challenges:
+
+![Challenge Tracts](./Images/Challenge_Flow_WTH.png)
+
+If you want to experience both User Flows and Custom Policies, then exercise each challenge in order.
+## Prerequisites
+
+- Your own Azure subscription with Owner access
+- Visual Studio Code
+- Azure CLI
+
+## Repository Contents
+
+- `../Coach`
+  - Coach's Guide and related files
+- `./Student/Resources/HarnessApp`
+  - Sample AspNetCore app to be used to interact with your B2C tenant
+- `./Student/Resources/MSGraphApp`
+  - Sample DotNetCore Console app to be used to query your B2C tenant
+- `./Student/Resources/Verify-inator`
+  - Sample AspNetCore WebApi app to be called by your B2C tenant's SignUp User Flows
+- `./Student/Resources/PageTemplates`
+  - Sample HTML page template that can be used to customize User Flows and Custom Policies
+- `../Student`
```

Comment:
Please move this to the top of the list above `../Coach`

Also I know it's nitpicky but can you change ./Student to ../Student for all the resources items.

Reply:
I changed references to ./Student since that would be the proper path